specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.0/examples/CopyModel.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "Content-Type": "application/json",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "modelId": "{modelId}",
+    "body": {},
+    "copyRequest": {
+      "targetResourceId": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CognitiveServices/accounts/{resourceName}",
+      "targetResourceRegion": "{region}",
+      "authorization": {
+        "modelId": "{modelId}"
+      }
+    }
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Operation-Location": "{endpoint}/formrecognizer/v2.0-preview/custom/models/{modelId}/copyResults/{resultId}"
```

Comment:
Will you have additional operations on models beside copy?
You may want to consider this pattern that exists in other services:

`{endpoint}/formrecognizer/v2.0-preview/custom/models/{modelId}/operations/{operationId}`
or
`{endpoint}/formrecognizer/v2.0-preview/custom/models/{modelId}/operations/copy/{operationId}`

Reply:
Hey Assaf,
 We do have the operations pattern in the FR. However, we already have a /analyseResults/{resultId} and I'm just following that pattern here with the /models/{modelId}/copyResults/{resultId} API.

Reply:
@assafi, could you share which other services use the pattern you describe?  Are there recommendations about that pattern in the MS or Cognitive REST API guidelines?  If there's an opportunity to standardize here across the Cognitive Services, it would be great to get that guidance in the Cognitive REST API Guidelines.